graderx/graders/stdout_graders/c/c_grader.py

```diff
@@ -89,15 +90,19 @@ def add_submissions(course, lab, submissions_file):
     submissions_file in the [lab_path/submissions/] directory  
     """
     lab_path = get_lab_path(course, lab)
-    lab_path_str = str(lab_path)
+    timestamp = helpers.current_timestamp()
     # extract_submissions will clean the target submissions directory before extracting
-    extract_submissions(Path(
-        f"{lab_path_str}/submissions"), submissions_file)
+    extract_submissions(lab_path.joinpath(f'submissions/{timestamp}'), submissions_file)
```

Comment:
please don't manually join paths using `/` use `os.path.join` or any equivalent instead. to keep the code cross platform

Reply:
I guess those joiner functions take unlimited number of arguments (variadic arguments), so you won't bother calling them multiple times

Reply:
okay noted, will fix that across all the app